### ISSUE TITLE
Pause/resume

### ISSUE THREAD

**Issue by vsevjednom-cz** [mendhak/gpslogger#823]
Example: I'm in shop center during run. Autopause not works here - when I'm near window. So I want to use manual pause 🖥️. Is it possible? 🙃

**Comment by mendhak** [comment]
What do you mean by autopause, I didn't understand that bit.  You can just hit the stop button to manually stop.

**Comment by vsevjednom-cz** [comment]
Autopause = I mean "do not record while I don't move" setting.

When i hit the stop button for manualy stop, I want to continue with the same GPX but when I come home and hit the stop button, it's the end of measure GPX. This is the reason why I need pause button.

**Comment by vsevjednom-cz** [comment]
Autopause = I mean "do not record while I don't move" setting.

When i hit the stop button for manually stop, I want to continue with the same GPX but when I come home and hit the stop button, it's the end of measure GPX. This is the reason why I need pause button.

**Comment by vsevjednom-cz** [comment]
Autopause = I mean "do not record while I don't move" setting.

When I hit the stop button for manually stop, I want to continue with the same GPX but when I come home and hit the stop button, it's the end of measure GPX. This is the reason why I need pause button.

**Comment by fatcat386** [comment]
> Autopause = I mean "do not record while I don't move" setting.

There are setting "don't log if still" exists in logging settings at the bottom. It work as it should if you not moving your device. Other possible choice is to set "distance before logging" to some value, so it will not record points if you staying in same position. Also there are option "custom file name" for log file, so maybe it is possible to write to same file after pressing stop and then start again. Maybe it somehow helps you but yeah there is no true "pause" equivalent.

**Comment by vsevjednom-cz** [comment]
Yes, your variant with custom name file works as I want but I don't want to have all my activities in one file. Pause/resume function has Strava and Sports Tracker. This is the only feature missing in the GPS Logger.
